docker/Dockerfile

```diff
@@ -307,18 +307,14 @@ COPY --from=csrc-build /workspace/dist /precompiled-wheels
 
 COPY . .
 
-ARG GIT_REPO_CHECK=0
-RUN --mount=type=bind,source=.git,target=.git \
-    if [ "$GIT_REPO_CHECK" != "0" ]; then bash tools/check_repo.sh ; fi
-
 ARG vllm_target_device="cuda"
 ENV VLLM_TARGET_DEVICE=${vllm_target_device}
 
 # Skip adding +precompiled suffix to version (preserves git-derived version)
 ENV VLLM_SKIP_PRECOMPILED_VERSION_SUFFIX=1
 
+ARG VLLM_VERSION_OVERRIDE="0.0.0+src.build"
```

Comment:
Defined in 
https://github.com/vllm-project/vllm/blob/ac1c934276167085dc7d9e5adb8c2089de0460f9/setup.py#L647-L653

Reply:
The parameter needs to be added when releasing the image.
Adding this parameter to the test will also cause the cache to miss.
```
--build-arg VLLM_VERSION_OVERRIDE=$(python3 -m setuptools_scm -f plain) 
```

Reply:
Can you include the release pipeline change in this PR so the picture full picture is clear?

Reply:
> Thanks [@wzshiming](https://github.com/wzshiming)
> 
> I think with this change we probably don't need the following step in the tests stage right ?
> 
> ```
> RUN echo 'tzdata tzdata/Areas select America' | debconf-set-selections \
>     && echo 'tzdata tzdata/Zones/America select Los_Angeles' | debconf-set-selections \
>     && apt-get update -y \
>     && apt-get install -y git
> ```

I've tested it and it doesn't work, the next step requires git

Reply:
yeah just noticed in the tests, thanks